packages/salesforcedx-vscode-replay-debugger/src/breakpoints/breakpointUtil.ts

```diff
@@ -36,6 +50,36 @@ export class BreakpointUtil {
     );
   }
 
+  public getRawLineBPInfo(): any | undefined {
+    return this.rawLineBPInfo;
+  }
+
+  public createMappingsFromLineBreakpointInfo(lineBpInfo: any): void {
```

Comment:
Are there tests for this?

Reply:
Yes. Those would be the tests above with the incorrect removal comment.